ansible_collections/arista/avd/docs/porting-guides/4.x.x.md

```diff
@@ -1,52 +1,59 @@
 # Porting Guide for Ansible AVD 4.x.x
 
 Major releases of AVD can contain breaking changes. The porting guide addresses how to update your inventory
-and playbooks to be compatible with new default behaviors and changed data models, when upgrading from AVD 3.x.x versions.
+and playbooks to be compatible with new default behaviors and changed data models when upgrading from AVD 3.x versions.
 
 Users of `eos_designs` do not have to consider the changes in `eos_cli_config_gen`, since those adaptions are
-built in to `eos_designs`.
+built into `eos_designs`.
 
 ## Changes to role `arista.avd.eos_designs`
 
 ### Network services variables
 
-- `evpn_rd_type` has been removed and replaced with `overlay_rd_type`.
-- `evpn_rt_type` has been removed and replaced with `overlay_rt_type`.
-- `vxlan_vlan_aware_bundles` has been removed and replaced with `evpn_vlan_aware_bundles`
+- Rename `evpn_rd_type` to `overlay_rd_type`
+- Rename `evpn_rt_type` to `overlay_rt_type`
+- Rename `vxlan_vlan_aware_bundles` to `evpn_vlan_aware_bundles`
 
-### P2P uplink mtu
+### PTP variables
```

Comment:
```suggestion
### Fabric variables
```

But TBH I would remove these headings since we only have a few keys in each.
The first one with the renames could be `### Renamed network services variables`